test/seeding.jl

```diff
@@ -31,52 +31,91 @@ k = 5
 X = rand(d, n)
 C = pairwise(SqEuclidean(), X)
 
+Xt = copy(transpose(X))
+Ct = copy(transpose(C))
+
 md0 = min_interdist(X)
 
 @testset "RandSeed" begin
+    Random.seed!(34568)
     iseeds = initseeds(RandSeedAlg(), X, k)
     @test length(iseeds) == k
     @test alldistinct(iseeds)
+    Random.seed!(34568)
```

Comment:
Is it really useful to repeat this? All that matters is that the result is the same across runs, and for this one call is enough.

Reply:
Yes I believe so given that the seeding functions are all random, the tape needs to be rewinded if I want to test that the same numbers are generated.